scss/components/_soe_global.scss

```diff
@@ -14,16 +14,19 @@ body {
   line-height: 1.55em;
 
   @include breakpoint-max(medium) {
+
```

Comment:
I know we want our convention to include spaces below mixins (@includes), but am not sure that this is required for breakpoint calls, unless you think otherwise. I might vote to note have a line break after these.